runtime/src/main/java/org/corfudb/runtime/collections/CorfuCounter.java

```diff
@@ -0,0 +1,26 @@
+package org.corfudb.runtime.collections;
+
+import org.corfudb.annotations.CorfuObject;
+import org.corfudb.annotations.DontInstrument;
+import org.corfudb.annotations.MutatorAccessor;
+
+@CorfuObject
+public class CorfuCounter {
```

Comment:
This needs to be checkpointed. Why not just use a map that's already supported?

Also, from what I understand multiple clients will be using this shared counter, so it seems like transactions are needed (i.e. need conflict parameters).

Reply:
this class will be removed in https://github.com/CorfuDB/CorfuDB/pull/1882